### DISCUSSION TITLE
Businesses need to participate in procurement

### DISCUSSION THREAD

**Discussion by eprocurementontology** [eprocurementontology/eprocurementontology#506] [Ideas]
Jáchym Hercher submitted the following use case to be further developed:
Businesses need to participate in procurement, i.e. follow the e-procurement phases described in the Project Charter (e.g. search opportunities and automatically receive suggestions; send expressions of interests, request to participate, tenders, tenders within mini-tenders, complaints, signed contracts; receive calls for tender in two-stage procedures, call-offs, information on whether they have been awarded a contract and how many points they received in the evaluation etc. etc.).

**Comment by muricna** [comment]
This is partly covered by use case 2
